lite/commit.go

```diff
@@ -2,98 +2,86 @@ package lite
 
 import (
 	"bytes"
-
-	"github.com/pkg/errors"
+	"errors"
+	"fmt"
 
 	"github.com/tendermint/tendermint/types"
-
-	liteErr "github.com/tendermint/tendermint/lite/errors"
 )
 
-// Certifier checks the votes to make sure the block really is signed properly.
-// Certifier must know the current set of validitors by some other means.
-type Certifier interface {
-	Certify(check Commit) error
-	ChainID() string
-}
-
-// Commit is basically the rpc /commit response, but extended
-//
-// This is the basepoint for proving anything on the blockchain. It contains
-// a signed header.  If the signatures are valid and > 2/3 of the known set,
-// we can store this checkpoint and use it to prove any number of aspects of
-// the system: such as txs, abci state, validator sets, etc...
-type Commit types.SignedHeader
-
-// FullCommit is a commit and the actual validator set,
-// the base info you need to update to a given point,
-// assuming knowledge of some previous validator set
+// FullCommit is a signed header (the block header and a commit that signs it),
+// the validator set which signed the commit, and the next validator set. The
+// next validator set (which is proven from the block header) allows us to
+// revert to block-by-block updating of lite certifier's latest validator set,
```

Comment:
Comment should not have historical connotation.

Reply:
I don't understand what you mean by historical connotation.  The past tense "signed" in "the validator which signed the commit" is correct.

Reply:
I meant this part "allows us to revert..."